Meta/HeaderCheck/generate_all.py

```diff
@@ -12,7 +12,9 @@
 
 
 def get_headers_here():
-    result = subprocess.run(['git', 'ls-files', 'Userland/Libraries/*.h'], check=True, capture_output=True, text=True)
+    result = subprocess.run(
```

Comment:
This seems like it should be in its own commit

Reply:
That feels like excessive splitting, but okay, will do :)